pkg/cron/cron.go

```diff
@@ -96,7 +96,8 @@ func notifyOnErr(err error, backoffDelay time.Duration) {
 // RequestCall - ZMS call for update crons
 func (c *Cron) requestCall() error {
 	master := false
-	domains, etag, err := c.zmsClient.GetSignedDomains("", "true", "", &master, c.etag)
```

Comment:
What is the `conditions` variable being used here for ?

Reply:
Hi @bhaskaramVaranasi , Please find the screenshot below.
This `conditions` field is one of the input parameter required to form the `url`. The value for this is kept as `false` since the other variable `master` was set to `false` in the same way we are setting this `conditions` field to `false`.

`Below screenshot is from cron.go`
<img width="955" alt="Screenshot 2024-06-05 at 1 11 02 PM" src="https://github.com/AthenZ/k8s-athenz-syncer/assets/65297307/a13a269b-f1e3-4443-86b9-0f68a2db48ec">

`Below screenshot is from client.go, this file is from the package athenz@v1.11.59/clients/go/zms/client.go`
![Screenshot 2024-06-03 at 10 17 01 AM](https://github.com/AthenZ/k8s-athenz-syncer/assets/65297307/9f2af5be-ffd1-417b-aba7-dd25ab2f09f9)